src/pages/bar.js

```diff
@@ -1,107 +1,40 @@
+import ingredientsStore from '../hooks/ingredientsStore';
 
 export default function BarPage() {
-	const fetchIngredients = useStore(state => state.fetchIngredients);
-	const searchItem = useStore(state => state.searchItem);
-	const setFilteredIngredients = useStore(state => state.setFilteredIngredients);
-	const ingredients = useStore(state => state.ingredients);
-	const filteredIngredients = useStore(state => state.filteredIngredients);
-	const updateIngredients = useStore(state => state.updateIngredients);
-	const [saveMode, setSaveMode] = useState(false);
+	const {data: session} = useSession();
+	const fetchSavedIngredients = ingredientsStore(state => state.fetchSavedIngredients);
+	const savedIngredients = ingredientsStore(state => state.savedIngredients);
 
 	useEffect(() => {
-		fetchIngredients('/api/mongoIngredients');
-	}, [fetchIngredients]);
-
-	useEffect(() => {
-		const items = ingredients.filter(ingredient =>
-			ingredient.name.toLowerCase().includes(searchItem.toLowerCase())
-		);
-		setFilteredIngredients(items);
-	}, [ingredients]);
-
-	useEffect(() => {
-		const items = ingredients.filter(ingredient =>
-			ingredient.name.toLowerCase().includes(searchItem.toLowerCase())
-		);
-
-		setFilteredIngredients(items);
-	}, [searchItem]);
+		if (session) {
+			fetchSavedIngredients(`/api/users/${session.user.email}`);
+		}
+	}, [fetchSavedIngredients, session]);
 
 	return (
 		<Layout>
 			<Head>
 				<title key="title">My Bar</title>
 				<meta key="description" name="description" content="About" />
 			</Head>
-			<>
-				<Search />
-				<IngredientGrid>
-					{filteredIngredients.map(ingredient => {
-						{
-							return saveMode || ingredient.saved ? (
-								<IngredientCard
-									color={ingredient.color}
-									mode={saveMode}
-									saved={ingredient.saved}
-									key={ingredient._id}
-								>
-									<h2>{ingredient.name}</h2>
-									{saveMode && !ingredient.saved ? (
-										<Button
-											onClick={() =>
-												updateIngredients({
-													...ingredient,
-													saved: true,
-												})
-											}
-										>
-											+
-										</Button>
-									) : (
-										<Button
-											onClick={() =>
-												updateIngredients({
-													...ingredient,
-													saved: false,
-												})
-											}
-										>
-											X
-										</Button>
-									)}
-								</IngredientCard>
-							) : undefined;
-						}
-					})}
-				</IngredientGrid>
-				{!saveMode ? (
-					<Button
-						onClick={() => {
-							setSaveMode(true);
-						}}
-					>
-						+
-					</Button>
-				) : (
-					<Button
-						onClick={() => {
-							setSaveMode(false);
-							setFilteredIngredients(ingredients);
-						}}
-					>
-						Done
-					</Button>
-				)}
-			</>
+			<IngredientGrid>
+				{!savedIngredients && <>There are no Ingredients in your Bar</>}
+				<Link href="/select">Add more Ingredients</Link>
```

Comment:
A anchor element is not allowed inside a `ul` element. [See caniinclude for further advice](https://caninclude.glitch.me/caninclude?child=a&parent=ul).